### PR TITLE
Given proper spacing before and after the footer tag

### PR DESCRIPTION
**Pull request by aashimgarg**
Issue no #95 
After this PR the user interface looks better.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -302,7 +302,7 @@ <h4><a href="./news/2019-07-02-Guelph-accessibility-conference.html">Floe at the
 
                 </div>
             </div>
-
+            <br><br>
```
Comment (jobara) [inline index.html]: Spaces should be done with CSS styles not using `<br>` tags. That is, the markup is to structure the content, CSS is to style it.

Comment (aashimgarg) [reply index.html]: ok i will make the changes!

**Comment by aashimgarg** [comment]
@jobara can you please review it?

**Comment by aashimgarg** [comment]
I made the foloowing changes @jobara !

**Comment by jobara** [comment]
@aashimgarg I left a [comment](https://github.com/fluid-project/floeproject.org/issues/95#issuecomment-590355628) on the issue asking for more information about what the issue actually it.

**Comment by sachin10101998** [comment]
Space is not required before the footer as it seems the continuity in the page gets broken. Should be closed along with #95

**Comment by greatislander** [comment]
This PR won't be merged as we are currently beginning a redesign process. Thanks for your contribution!

**Pull request closed without merging**